src/CommonLib/hostfxr_utility.cpp

```diff
@@ -308,4 +317,4 @@ HOSTFXR_UTILITY::SetHostFxrArguments(
         DBG_ASSERT(pwzArgs == NULL);
     }
     return hr;
-}
\ No newline at end of file
+}
```

Comment:
in case of failure, you need to call SysFreeString to free memory used by argv otherwise there will be memory leak